gslib/commands/signurl.py

```diff
@@ -128,7 +129,12 @@
                the duration the link remains valid is the sum of all the
                duration options.
 
-               The max duration allowed is 7d.
+               The max duration allowed is 7d when private-key-file is used.
+
+               The max duration allowed is 12 hours when -u option is used.
+               This limitation exists because the system managed key used for
```

Comment:
"system-managed key used to sign the url"

Reply:
Done